web/src/engine/main/src/keymanEngine.ts

```diff
@@ -196,6 +196,9 @@ export default class KeymanEngine<
 
     config.initialize(optionSpec);
 
+    // Initialize supplementary plane string extensions
+    String.kmwEnableSupplementaryPlane(true);
```

Comment:
This turns on non-BMP string processing in all our app/targets -- not just web browsers, but also when embedded in iOS and Android apps.